service/src/main/java/greencity/service/EcoNewsCommentServiceImpl.java

```diff
@@ -256,7 +259,8 @@ public int countReplies(Long id) {
      */
     @Override
     public int countOfComments(Long ecoNewsId) {
-        return ecoNewsCommentRepo.countOfComments(ecoNewsId);
+        return ecoNewsCommentRepo.countEcoNewsCommentByEcoNews(ecoNewsRepo.findById(ecoNewsId)
```

Comment:
Could you please write the additional test for the controller, where you handle this Exception ?